index.js

```diff
@@ -441,6 +441,20 @@
   //. ```
   Maybe.prototype.of = def('Maybe#of', [a], Maybe.of);
 
+  //# Maybe#reduce :: Maybe a ~> (b -> a -> b) -> b -> b
+  //.
+  //. Takes a function and an initial value, returning the result of applying
+  //. the function to the initial value and the value of the Just. If `this`
+  //. is a Nothing, then the initial value is simply returned instead.
```

Comment:
I typically describe it as an *initial accumulator value*, but there isn't a whole lot of accumulating going on in the implementation for Maybe.